images/bootstrap/runner

```diff
@@ -37,6 +37,8 @@ cleanup_dind() {
     echo "Remaining docker images and volumes are:"
     docker images --all || true
     docker volume ls || true
+    echo "Cleaning up bimft_misc ..."
+    (set -x; echo -1 >/proc/sys/fs/binfmt_misc/status || true)
```

Comment:
is it `-1` or `0` for resetting? https://access.redhat.com/solutions/1985633 seems to use 0

Reply:
referenced docs now